app/helpers/home_helper.rb

```diff
@@ -0,0 +1,2 @@
+module HomeHelper
+end
```

Comment:
Not sure if this was auto-generated, but can we remove it since it doesn't do anything?